apps/storybook/src/forms/Checkbox.stories.tsx

```diff
@@ -15,11 +15,7 @@ export const Basic: Story = {};
 
 export const HelperText: Story = {
   args: {
-    endDecorator: (
-      <Text color="fg.secondary" fontSize="sm">
-        Helper Text
-      </Text>
-    ),
+    endDecorator: <Text>Helper Text</Text>,
```

Comment:
technically we can just change these to strings
```suggestion
    endDecorator: "Helper Text",
```